spec/values/calculation/sign.hrx

```diff
@@ -0,0 +1,141 @@
+<===> README.md
+Most of the same behavior tested for `calc()` applies to `sign()`, but for
+terseness' sake isn't tested explicitly.
+
+<===>
+================================================================================
+<===> positive/input.scss
+a {b: sign(3)}
+
+<===> positive/output.css
+a {
+  b: 1;
+}
+
+<===>
+================================================================================
+<===> negative/input.scss
+a {b: sign(-5.6)}
+
+<===> negative/output.css
+a {
+  b: -1;
+}
+
+<===>
+================================================================================
+<===> zero/input.scss
+a {b: sign(0)}
```

Comment:
This should divide 1 by the result so we can verify that it's actually _positive_ zero, since this can sometimes return negative zero.